specs/wp-signup-spec.js

```diff
@@ -305,6 +305,216 @@ describe( `[${ host }] Sign Up  (${ screenSize }, ${ locale })`, function() {
 		} );
 	} );
 
+	describe.only( 'Sign up for a non-blog site on a premium paid plan through main flow in USD currency using a coupon @parallel @visdiff', function() {
+		const blogName = dataHelper.getNewBlogName();
+		const expectedBlogAddresses = dataHelper.getExpectedFreeAddresses( blogName );
+		const emailAddress = dataHelper.getEmailAddress( blogName, signupInboxId );
+		const currencyValue = 'USD';
+		const expectedCurrencySymbol = '$';
+		let originalCartAmount;
+
+		before( function() {
+			let testEnvironment = 'WordPress.com';
+			let testName = `Signup [${ global.browserName }] [${ screenSize }]`;
+			eyesHelper.eyesOpen( driver, eyes, testEnvironment, testName );
+		} );
+
+		before( async function() {
+			return await driverManager.ensureNotLoggedIn( driver );
+		} );
+
+		step( 'We can set the sandbox cookie for payments', async function() {
```

Comment:
coupons won't work sandboxed - so we shouldn't set the sandbox, but we won't also complete the purchase